tests/app/dao/test_services_dao.py

```diff
@@ -380,6 +384,44 @@ def test_get_all_user_services_should_return_empty_list_if_no_services_for_user(
     assert len(dao_fetch_all_services_by_user(user.id)) == 0
 
 
+@freeze_time('2019-04-23T10:00:00')
+def test_dao_fetch_live_services_data(sample_user, mock):
+    org = create_organisation()
+    service = create_service(go_live_user=sample_user)
+    template = create_template(service=service)
+    service_2 = create_service(service_name='second', go_live_user=sample_user)
+    create_service(service_name='third')
+    create_service(service_name='not_live', count_as_live=False)
+    template2 = create_template(service=service, template_type='email')
+    template_letter_1 = create_template(service=service, template_type='letter')
+    template_letter_2 = create_template(service=service_2, template_type='letter')
+    dao_add_service_to_organisation(service=service, organisation_id=org.id)
+    create_ft_billing(bst_date='2019-04-20', notification_type='sms', template=template, service=service)
+    create_ft_billing(bst_date='2019-04-21', notification_type='sms', template=template, service=service)
+    create_ft_billing(bst_date='2018-04-20', notification_type='sms', template=template, service=service)
```

Comment:
@servingUpAces this ft_billing is dated at 2018 so it does not appear in the final result, is this sufficient as a test?